tests/crypto/test_crypto.c

```diff
@@ -7,7 +7,10 @@
 #include <unistd.h>
 
 #include "blake2b_data.h"
+#include "core/utils/byte_buffer.h"
+#include "core/utils/macros.h"
```

Comment:
remove unused headers. 
`HEX_TO_BIN_BYTES` is not wildly used, it can be defined here, not in `core/utils/macros.h`
```suggestion

```